tests/fw_rules_tests/policies/expected_output/istio-deny-all-scheme_output.txt

```diff
@@ -1,6 +1,6 @@
 final fw rules for query: istio-deny-all, config: istio-deny-all:
-src ip block: 0.0.0.0/0 dst_ns: [kube-system,vendor-system] dst_pods: [*] conn: TCP 1-65536,
-src ip block: ::/0 dst_ns: [kube-system,vendor-system] dst_pods: [*] conn: TCP 1-65536,
-src_ns: [default,kube-system,vendor-system] src_pods: [*] dst_ns: [kube-system,vendor-system] dst_pods: [*] conn: TCP 1-65536,
+src ip block: 0.0.0.0/0 dst_ns: [kube-system,vendor-system] dst_pods: [*] conn: TCP All,
+src ip block: ::/0 dst_ns: [kube-system,vendor-system] dst_pods: [*] conn: TCP All,
```

Comment:
If possible, please avoid printing `All` for ports. `conn: TCP` will be enough. This will also be consistent with the yaml output

Reply:
done